lib/charms/layer/apache_kafka.py

```diff
@@ -14,14 +16,29 @@ def __init__(self, dist_config=None):
         self.resources = {
             'kafka': 'kafka-%s' % utils.cpu_arch(),
         }
-        self.verify_resources = utils.verify_resources(*self.resources.values())
+
+    def verify_resources(self):
```

Comment:
Ok, so we had the discussion and I understand that the 0-byte resource is to make it optional from the admin but to fall back to non-resource behavior if it's not provided.  I don't want to do that in this case, since I'd like to phase out the S3 bucket entirely in favor of 2.0-style resources.